python/cudf_polars/cudf_polars/testing/asserts.py

```diff
@@ -21,8 +21,9 @@
 
 
 # Will be overriden by `conftest.py` with the value from the `--executor`
-# command-line argument
+# and `--scheduler` command-line argument
 Executor = None
+Scheduler = "synchronous"
```

Comment:
This being named `Scheduler` caused me a moment of confusion. Maybe `SCHEDULER` or `DEFAULT_SCHEDULER`? Although `Scheduler` is consistent with `Executor`.

Reply:
Okay, yeah. I renamed `Executor` and `Scheduler` to `DEFAULT_EXECUTOR` and `DEFAULT_SCHEDULER` to clarify things a bit (I hope).